### PR TITLE
Add Comment content update functionality 

### PR DESCRIPTION
**Pull request by ArpitBlagan**
### PR Fixes:
- 1 Add Comment Content Update Functionality
Link: https://www.loom.com/share/31e290bdcc2b4c3ca9894174c74bb058?sid=158fdd30-99a8-403e-a0ba-617745ec201f
### Checklist before requesting a review
- [✅] I have performed a self-review of my code
- [✅] I assure there is no similar/duplicate pull request regarding same issue

### CODE REVIEW

File: .pnp.cjs
Comment (siinghd) [inline .pnp.cjs:1]: remove this

File: .pnp.loader.mjs
Comment (siinghd) [inline .pnp.loader.mjs:1]: remove this

File: yarn.lock
Comment (siinghd) [inline yarn.lock:1]: fix this or remove this

**Review by siinghd — CHANGES_REQUESTED** [review]

File: yarn.lock
Comment (ArpitBlagan) [reply yarn.lock:1]: Found a bug as I am using dialog inside the dropdown so after fixing that I have created a new pr Added comment updating functionality #813

**Pull request closed without merging**